pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -1,86 +1,99 @@
+#!bash
```

Comment:
And besides, there's no point in having a shebang because this script doesn't so anything. All it does is define functions, so it must necessarily be `source`d. Doing "./make-wrapper.sh" makes no sense.